docs_nnx/guides/gemma.ipynb

```diff
@@ -82,13 +90,21 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "If everything went well, you should see:\n",
```

Comment:
Adding an extra optional step here, similar to what we have in the Gemma docs.

"**Note:** In Google Colab, you can instead authenticate into Kaggle using the code below after following the optional step 3 from above...."